src/lessons/02_introduction_to_frontend/introduction-to-frontend.slides.html

```diff
@@ -0,0 +1,688 @@
+            </ul>
+
+            <aside class="notes">
+              <ul>
+                <li>HTML is adding content in a way of organising them by adding structures around</li>
+                <li>Explain browser reading</li>
+                <li>Explain who actually defines guidelines and conventions</li>
+                <li>International community to organise guidelines around content strucuture in webpages. The World Wide Web Consortium is the main international standards organization for the World Wide Web. (They work with versions as well)</li>
+                <li>We're in the version 5 of HTML</li>
+              </ul>
+            </aside>
+          </section>
+        </section>
+
+        <!--SLIDE 4-->
+        <!--Introduction to HTML (basic structure - recap)-->
+        <section>
+          <!--SLIDE 4.1-->
+          <section data-auto-animate data-timing="10">
+            <h2>Introduction to HTML</h2>
+            <p>Basic Structure</p>
+          </section>
+
+          <!--SLIDE 4.2-->
+          <section data-auto-animate data-timing="10">
+            <h2>Introduction to HTML</h2>
+            <p>Basic Structure</p>
+
+            <pre data-auto-animate><code data-trim class="html"><script type="text/template">
+              <!DOCTYPE html>
+              <html>
+                <head>
+                  <title>My first website</title>
+                </head>
+                <body>
+                  <h1>Welcome!</h1>
+                  <p>This is my first HTML site.</p>
+                </body>
+              </html>
+            </script></code></pre>
+
+            <img data-src="images/skip.png" class="fragment fade-in"/>
+          </section>
+        </section>
+
+        <!--SLIDE 5-->
+        <!--Introduction to HTML (basic structure - following with class)-->
+        <section data-timing="600">
+          <h2>Introduction to HTML</h2>
+          <p>Basic Structure</p>
+
+          <aside class="notes">
+            Remind troupe we don't need everyone to code at this point
+          </aside>
+
+          <section data-auto-animate>
+            <pre data-id="code-animation"><code data-trim class="html"><script type="text/template">
+              <!DOCTYPE html>
+              <html>
+                <head>
+                  <title>My first website</title>
+                </head>
+                <body>
+                  <h1>Welcome!</h1>
+                  <p>This is my first HTML site.</p>
+                </body>
+              </html>
+            </script></code></pre>
+          </section>
+          <section data-auto-animate>
+            <pre data-id="code-animation"><code data-trim class="html"><script type="text/template">
+              <!DOCTYPE html>
+            </script></code></pre>
+
+            <p style="font-size: .3em;" class="current-visible">
+              👉 The <i>DOCTYPE</i> declaration is an instruction to the web browser about what version of HTML the page is written in.
```

Comment:
Can we make this font a larger? Is a bit hard to read. Same for the other similar elements

Reply:
I agree it is a bit hard to read